services/galley/src/Galley/API/Internal.hs

```diff
@@ -318,6 +332,14 @@ iPutTeamFeature ::
   Galley r (Public.TeamFeatureStatus a)
 iPutTeamFeature setter = Features.setFeatureStatus @a setter DontDoAuth
 
+iUnlockPayment ::
```

Comment:
not sure at this point, but just to not forget this: we may want to make this `iLockOrUnlock`, and pass a `Public.PaymentStatus` as an extra argument.  then we don't need to write `iLockPayment`.  Same probably goes for other places further down the call graph like the function to write into cassandra etc.